src/board_controller/pieeg/inc/periphery/CHANGELOG.md

```diff
@@ -0,0 +1,184 @@
+* v2.4.2 - 07/05/2023
```

Comment:
c_periphery should be in third_party folder

Reply:
c_periphery moved to third_party folder and build.cmake updated.